src/core/TypescriptConfig.ts

```diff
@@ -82,15 +105,98 @@ export class TypescriptConfig {
 			}
 		}
 	}
+
+	public categorizeDiagnostics(diagnostics: ReadonlyArray<ts.Diagnostic> | Readonly<ts.Diagnostic>): ICategorizedDiagnostics {
```

Comment:
how can I test this in action?

Reply:
In playground, modify `tsconfig.json` file with invalid settings such as `{ "target": "asd", "jsx": "asd" }`